Nfield.Quota/Persistence/QuotaFrameEncoder.cs

```diff
@@ -18,7 +18,8 @@ public static string Encode(QuotaFrame frame)
 
             resolver.Ignore(
                 typeof(QuotaFrameLevel),
-                nameof(QuotaFrameLevel.Target));
+                nameof(QuotaFrameLevel.Target),
```

Comment:
Is Target MinTarget? Should we rename this, maybe?

Reply:
We decided to keep the old name for min target